packages/langium/src/lsp/reference-finder.ts

```diff
@@ -60,14 +76,11 @@ export class DefaultReferenceFinder implements ReferenceFinder {
                     refs.push({ docUri: document.uri, range: reference.$refNode.range });
                 } else {
                     const range = reference.segment.range;
-                    refs.push({ docUri: reference.sourceUri, range });
+                    refs.push({ docUri: reference.sourceUri, range: range });
```

Comment:
Minor: just `range` is enough, or `range: reference.segment.range`

Reply:
Personal taste, I find it "cleaner" to have an explicit name for properties when declaring an object.

Reply:
I will not cry, it is not that important to me. It is just sugar from TypeScript.

Reply:
That will be removed because the `Location` type as a `create` function